ChangeLog

```diff
@@ -1,3 +1,58 @@
+3.18.2:
+	- 'N' or 'Ns' signal specs can now be used to sleep
+	  between signals sent by 'processes' promises
+	  (CFE-2207, ENT-5899)
+	- Added a test leveraging variablesmatching() to check that the agent
+	  doesn't crash when variables defined from a module do not have
+	  explicit tags (ENT-7724)
+	- Added bootstrap/failsafe exclusion for directories named .no-distrib
+	  (ENT-8079)
+	- Added protocol 3 (cookie) to syntax description (ENT-8560)
+	- cf-secret encrypt now encrypts for localhost if no key or host is
+	  specified (CFE-3874)
+	- CFEngine processes now try to use getent if the
+	  builtin user/group info lookup fails (CFE-3937)
+	- Custom promise modules now never get promise data with unresolved variables
+	  (CFE-3434)
+	- Custom promises now use standard promise locking and support ifelapsed
+	  (CFE-3434)
+	- Federated Reporting is now supported on RHEL 8 hubs (ENT-7493)
+	- Fixed wrong use of log level in users promises log messages (CFE-3906)
+	- Fixed default for ignore_missing_bundles and ignore_missing_inputs
+	  The issue here was that these attributes should default to false,
+	  but when they are assigned with an unresolved variable, they
+	  would default to true. (ENT-8430)
+	- Fixed inconsistencies with methods promises and missing bundles
```

Comment:
Can we summarize this one? Seems awfully long for the changelog. The bummer is that the details aren't easily found from the ticket since it is an ENT ticket.

Reply:
We could clone the ENT to CFE stripping anything we need to and then this large chunk could be in that ticket.